src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.setAttribute("drivers", driverService.getAll());
+        req.setAttribute("cars", carService.getAll());
+        req.getRequestDispatcher("/WEB-INF/views/addDriverToCar.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = Long.valueOf(req.getParameter("driver[]"));
+        Long carId = Long.valueOf(req.getParameter("carId"));
```

Comment:
You should use one style
```suggestion
        Long driverId = Long.valueOf(req.getParameter("driver_id"));
        Long carId = Long.valueOf(req.getParameter("car_id"));
```
![image](https://user-images.githubusercontent.com/20326858/227901187-ca94faff-1e3a-4f59-b9d5-69f959c2c9bf.png)